tests/console/znc.pm

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Setup ZNC IRC bouncer
+# Maintainer: Dominik Heidler <dheidler@suse.de>
+
+use base 'consoletest';
+use strict;
+use testapi;
+use utils qw(zypper_call systemctl);
+
+sub run {
+    select_console('root-console');
+    zypper_call('in znc');
+
+    script_run("sudo -u znc znc --makeconf; echo zncconf-status-\$? > /dev/$serialdev", 0);
+    sleep 1;
```

Comment:
Please get rid of all sleeps and replace by other sync methods. We have good examples in other test modules

Reply:
removed the sleeps